code/modules/mob/living/living.dm

```diff
@@ -10,6 +10,7 @@
 	faction += "[REF(src)]"
 	GLOB.mob_living_list += src
 	SSpoints_of_interest.make_point_of_interest(src)
+	update_fov()
```

Comment:
why does this not dispatch to the fov handler

Reply:
Instantiated mob might not get the handler because it may not have a client. Determining a thing's FOV despite it being clientless could be important for any expansions upon the system, but currently it isn't important and could be updated when the handler is about to be created, I can change it if need be.